ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/AddItemsToPortal/AddItemsToPortal.cpp

```diff
@@ -195,8 +195,15 @@ void AddItemsToPortal::connectUserSignals()
 
     if (m_user->items()->size() > 0)
```

Comment:
I think the "size > 0" check is now redundant and can be removed -- the for loop will handle the "size=0" case by doing nothing.

Reply:
Good call, thanks!